src/ts/components/watermark.ts

```diff
@@ -18,6 +18,8 @@ export class Watermark extends ClickOverlay {
     this.config = this.mergeConfig(config, {
       cssClass: 'ui-watermark',
       url: 'http://bitmovin.com',
+      role: 'img',
+      text: 'bitmovin-logo'
```

Comment:
In a forked UI it does not need to be the `bitmovin-logo`

Reply:
We should have a text content inside of the watermark button for the accessibility validation. Not sure what text we should put here

Reply:
just `logo`?